src/components/Inventory/view/InventorySeedlingsTable.tsx

```diff
@@ -232,17 +239,19 @@ export default function InventorySeedslingsTable(props: InventorySeedslingsTable
 
   return (
     <Grid item xs={12} sx={{ marginTop: theme.spacing(1) }}>
-      <BatchDetailsModal
-        open={openNewBatchModal}
-        reload={reloadData}
-        onClose={() => {
-          onUpdateOpenBatch(null);
-          setOpenNewBatchModal(false);
-        }}
-        organization={organization}
-        speciesId={speciesId}
-        selectedBatch={selectedBatch}
-      />
+      {openNewBatchModal && (
+        <BatchDetailsModal
+          open={openNewBatchModal}
```

Comment:
Since `open={openNewBatchModal}` why conditionally render this?

Reply:
We have some overloaded/complex modals that are hard to clean up on close. This conditional rendering allows for a clean instance every time we add a new batch. Not ideal.

Reply:
fixed it

Reply:
🎉 thanks!